pipenv/core.py

```diff
@@ -103,7 +103,7 @@
     STARTING_LABEL = '   '
 # Enable shell completion.
 click_completion.init()
-# Disable colors, for the soulless.
+# Disable colors, for the color blind and others who do not prefer colors.
 if PIPENV_COLORBLIND:
     crayons.disable()
 # Disable spinner, for cleaner build logs (the unworthy).
```

Comment:
I admit it looks like an unneeded change. The "crayons.disable()", however, gets executed after process.py is already imported and, since the BAR_FILLED_CHAR and BAR_EMPTY_CHAR are set up as a global variables in process.py, they already have the ANSI color code wrappers embedded in them.

If you are of opinion this is not needed and need a proof, then do

    export PIPENV_COLORBLIND=1
    mkdir pr2030 && cd pr2030
    pipenv install requests
    
The only thing in color will be the progress bar.